include/cpu.h

```diff
@@ -1,8 +1,8 @@
 #ifndef _CPU_H
 #define _CPU_H
 
-#include <stdint.h>
 #include "version.h"
+#include <stdint.h>
```

Comment:
I think it's kinda weird that clang-format puts system headers below project headers, I think the way it was before looked better. Maybe add a newline to separate them into two groups, like in src/sys.c?